pkg/api/mlflow/service/run/service.go

```diff
@@ -73,6 +73,7 @@ func NewService(
 func (s Service) CreateRun(
 	ctx context.Context, ns *models.Namespace, req *request.CreateRunRequest,
 ) (*models.Run, error) {
+	adjustCreateRunRequestForNamespace(ns, req)
```

Comment:
Im not sure about this function at all. we already have `ConvertCreateRunRequestToDBModel` which converts actual input `request` into `db` model, so for me it looks reasonable to put logic into `ConvertCreateRunRequestToDBModel` instead of have another function.

Reply:
I thought about this and it's a good suggestion, but I thought it would be best to make the step very explicit with a separate named method.

Reply:
additionally, in this function there is a reliance on finding the correct experiment to provide as a parameter to `ConvertCreateRunRequestToDBModel` so I needed to make the adjustment before entering the converter. If we move this adjustment inside the Converter, and it lets us do this adjustment in a uniform way across functions (others do not have ConvertToDBModel).

Reply:
This seems like a reasonable need to have a separate function.